pkg/api/latest/latest.go

```diff
@@ -76,6 +80,20 @@ func (g GroupMetaMap) GroupOrDie(group string) *GroupMeta {
 	return groupMeta
 }
 
+// AllPreferredGroupVersions returns the preferred versions of all registered
+// groups in the form of "group1/version1,group2/version2,..."
+func (g GroupMetaMap) AllPreferredGroupVersions() string {
+	if len(g) == 0 {
+		return ""
+	}
+	var defaults string
+	for _, groupMeta := range g {
+		defaults += groupMeta.GroupVersion + ","
+	}
+	defaults = strings.TrimRight(defaults, ",")
```

Comment:
Sure. I'm just trying to improve the efficiency by skipping the sort, as we don't need a deterministic order except for test.